plane/src/database/connect.rs

```diff
@@ -156,6 +161,17 @@ async fn create_backend_with_key(
         }
     };
 
+    sqlx::query!(
+        r#"
+        insert into backend_state (backend_id, state, created_at)
```

Comment:
I think we should use the same timestamp here as we use for `last_status_time` above

Reply:
Because they are both in the same transaction, they both refer to the same timestamp ([doc](https://www.postgresql.org/docs/9.3/functions-datetime.html#FUNCTIONS-DATETIME-CURRENT))

> `now()` is a traditional PostgreSQL equivalent to `transaction_timestamp()`.

Reply:
TIL